gramming/readme.txt

```diff
@@ -0,0 +1,31 @@
+=== Gramming ===
+Contributors: Themes Design
+Requires at least: 5.8
+Tested up to: 5.9
+Requires PHP: 5.7
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+== Description ==
+
+Bring the Social Media experience to your site and transform it into a repository for your photos and videos. Get Gramming now.
+
+== Changelog ==
+
+= 0.0.1 =
+* Initial release
+
+== Copyright ==
+
+Gramming WordPress Theme, (C) 2023 Themes Design
+Gramming is distributed under the terms of the GNU GPL.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
```

Comment:
We should add the credits for the `man-behind-the-camera-scaled.jpg` image, and possibly the images used in the screenshot too.